tools/bouncer_worker/src/queues/modelQueueHandler.js

```diff
@@ -54,7 +56,30 @@ Handler.onMessageReceived = async (cmd, rid, callback) => {
 	};
 
 	try {
-		returnMessage.value = await runBouncerCommand(logDir, cmdParams);
+		const memoryReporting = { enabled: config.elastic.memoryReporting, maxMemory: 0, processTime: 0 };
+		returnMessage.value = await runBouncerCommand(logDir, cmdParams, memoryReporting);
+		if (memoryReporting.maxMemory > 0) {
+			const importFile = cmdParams[3];
+			// eslint-disable-next-line
+			const fileStats = fs.statSync(require(importFile).file);
+			const elasticBody = {
+				Teamspace: user,
```

Comment:
teamspace should be the same as database.

user is the person who uploaded the file. database is where it's going to.

e.g. if `carmen` uploads a file in `keif`'s teamspace. user = `carmen`, teamspace & database = `keif`